lib/services.template.ejs

```diff
@@ -315,9 +315,14 @@ module
     // Note: LocalStorage converts the value to string
     // We are using empty string as a marker for null/undefined values.
     function save(storage, name, value) {
-      var key = propsPrefix + name;
-      if (value == null) value = '';
-      storage[key] = value;
+      try{
+        var key = propsPrefix + name;
+        if (value == null) value = '';
+        storage[key] = value;
+      }
+      catch(err){
+          console.log('Cannot access local/session storage:', err);
```

Comment:
1) `catch` goes on the same line as the closing bracket
2)  Indentation - use exactly 2 spaces per indentation level.

```js
} catch(err) {
  console.log(...);
}
```